client/src/pages/EditAccount.js

```diff
@@ -0,0 +1,180 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+
+// dummy data props,context, redux etc
+const firstName = "Cees";
+const lastName = "Wang";
+const selfIntroduction =
+  "  Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diamnonummy nibh euismod tincidunt ut laoreet dolore magna";
+const facebookURL = "http://facebook.com";
+const twitterURL = "http://twitter.com";
+const githubURL = "http://github.com";
+const linkedinURL = "http://linkedin.com";
+const personalURL = "http://personal.com";
+
+const editAccount = true;
+
+function getInitials(firstName, lastName) {
```

Comment:
I need this function too for Posts! I'll make this a common function we can use. I'll create a file called `userProfile.js` in `client/src/utils` and export this function in there.